### PR TITLE
Extract forms into their own components and fix support us cards

### PR DESCRIPTION
**Pull request by ZBeimnet**

### CODE REVIEW

File: Frontend/pages/support.vue [hunk 1]
```diff
@@ -235,75 +110,28 @@ export default {
...
-          'Experience must be less than 150 characters',
-      ],
-      storyRules: [
-        (v) => !!v || 'Story is required',
-        (v) =>
-          (v && v.length <= 150) || 'Story must be less than 150 characters',
-      ],
+      Interviews: false,
+      Mentorship: false,
+      'Q & A': false,
```
Comment (TripleThreads) [inline Frontend/pages/support.vue:123]: why didn't you use a single variable that stores in string? for example form_type: "interview"

**Review by TripleThreads — CHANGES_REQUESTED** [review]

File: Frontend/pages/support.vue [hunk 2]
```diff
@@ -44,154 +44,29 @@
...
+      </div>
     </v-container>
   </div>
 </template>
 
 <script>
 import Banner from '@/components/Core/Banner'
+import InterviewForm from '@/components/Support/InterviewForm'
+import MentorshipForm from '@/components/Support/MentorshipForm'
+import QandAForm from '@/components/Support/QandAForm'
```
[the remark this answers is not in the document]
Comment (ZBeimnet) [reply Frontend/pages/support.vue]: Yeah, this is a better way. Thanks!

File: Frontend/pages/support.vue [hunk 3]
```diff
@@ -7,7 +7,7 @@
       :description="banner.description"
     />
 
-    <v-container style="margin-top: -5%">
+    <v-container style="margin-top: -75px">
```
Comment (TripleThreads) [inline Frontend/pages/support.vue:10]: I wonder what this looks on mobile :)

File: Frontend/pages/support.vue [hunk 4]
```diff
@@ -46,145 +46,13 @@
...
-              <v-icon class="ml-2" small />
-            </v-btn>
-          </div>
-        </v-form>
-      </v-card>
+      <div id="form_card">
+        <InterviewForm v-if="form_type === 'Interviews'" />
+        <MentorshipForm v-if="form_type === 'Mentorship'" />
+        <QandAForm v-if="form_type === 'Q & A'" />
+      </div>
```
Comment (TripleThreads) [inline Frontend/pages/support.vue:55]: I would like this in methods.

**Review by TripleThreads — APPROVED** [review]

**Comment by TripleThreads** [comment]
All issues are addressed.

**Pull request merged**